package-lock.json

```diff
@@ -5,9 +5,9 @@
   "requires": true,
   "dependencies": {
     "@okta/okta-auth-js": {
-      "version": "1.13.0",
-      "resolved": "https://registry.npmjs.org/@okta/okta-auth-js/-/okta-auth-js-1.13.0.tgz",
-      "integrity": "sha1-9KX3Oku+MquqULuYWkp0rjnHYkg=",
+      "version": "1.17.0",
+      "resolved": "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta/@okta/okta-auth-js/-/@okta/okta-auth-js-1.17.0.tgz",
```

Comment:
this is wrong, it should be the public npm registry link

Reply:
Done